.pyup.yml

```diff
@@ -1,4 +1,7 @@
-# autogenerated pyup.io config file 
+# autogenerated pyup.io config file
 # see https://pyup.io/docs/configuration/ for all available options
 
 update: insecure
+requirements:
+  - Pipfile
+  - pipenv.txt
```

Comment:
There is no pipenv.txt

Reply:
Sorry, yeah, there is/will be, over in https://github.com/mozilla/servicebook/pull/61 - I'll remove this here, as I was intending to make these changes clean and separate.

Reply:
Fixed.